universal-application-tool-0.0.1/.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('gts/.prettierrc.json'),
+}
```

Comment:
suggestion (blocking): happy to have more autoformatting for JS/TS! would prefer it to be in a separate PR though